GEOSldas_App/setup_utils.py

```diff
@@ -0,0 +1,309 @@
+#!/usr/bin/env python3
+
+import os
+import sys
+from collections    import OrderedDict
+
+def parseInputFile(inpfile, ladas_cpl=0):
+   """ 
+     Parse the input file and return a dict of options
+     Inpfile : input file
+     Output  : dict, if inpfile does not exist, return empty {}
+   """ 
+   if not os.path.exists(inpfile):
+     assert ladas_cpl > 0, " No exeinput file only if ladas_cpl > 0"
+     return {}
+        
+   inpdict = OrderedDict()
+   errstr  = "line [%d] of [%s] is not in the form 'key: value'"
+   # determine which default values to pick from GEOS_SurfaceGridComp.rc
+   # must be in the format '# GEOSldas=>' or # GEOSagcm=>
+   if ladas_cpl == 0 :
+      use_rc_defaults = '# GEOSldas=>'    # use defaults for LDAS
+   else :
+      use_rc_defaults = '# GEOSagcm=>'    # use defaults for AGCM
```

Comment:
I think using the '#' character and space as part of the string is risky.  Future edits of GEOS_SurfaceGridcomp.rc may inadvertently drop the blank character after the '#' character, which would break the present script.  Rather than starting the line with '# GEOS...', why not identify the lines that contain 'GEOS....=>' as we did before?